fighthealthinsurance/encrypted_amount_field.py

```
@@ -0,0 +1,63 @@
+
+    description = "An encrypted scalar string (Fernet ciphertext base64)."
+
+    def from_db_value(
+        self, value: Optional[str], expression: Any, connection: Any
+    ) -> str:
+        if value is None or value == "":
+            return ""
+        try:
+            decrypted: bytes = Cryptographer.decrypted(value.encode("utf-8"))
+            return decrypted.decode("utf-8")
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)